lms/djangoapps/courseware/testutils.py

```diff
@@ -148,9 +148,9 @@ def verify_response(self, expected_response_code=200, url_params=None):
         return response
 
     @ddt.data(
-        ('vertical_block', ModuleStoreEnum.Type.mongo, 14),
+        ('vertical_block', ModuleStoreEnum.Type.mongo, 10),
         ('vertical_block', ModuleStoreEnum.Type.split, 6),
-        ('html_block', ModuleStoreEnum.Type.mongo, 15),
+        ('html_block', ModuleStoreEnum.Type.mongo, 11),
         ('html_block', ModuleStoreEnum.Type.split, 6),
     )
```

Comment:
This counts how many mongo calls are used in different scenarios, they will decrease since less code is running now.